ca_tools/CMakeLists.txt

```diff
@@ -1,15 +1,29 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(ca_tools)
 
-find_package(catkin REQUIRED)
+find_package(catkin REQUIRED COMPONENTS actionlib_msgs)
+
+add_action_files(
+  DIRECTORY action
+  FILES linefollower.action
```

Comment:
If you add the directory action, you don't need to manually add the `linefollower.action` file.
Also, check if you're following the ROS convention.

Reply:
Deleted `FILES` line.